### PR TITLE
Channel progress bars

### PR DESCRIPTION
**Pull request by PatrickLemke**
This pull request adds channel progress bars to the channel overview.

Let me know what you think, I can make changes to the style if needed. This also adds Bootstrap to the Project so if this is not wanted, I can try to accomplish the task in a different way.

There is still one bug in this which I don't understand:
When the plugin is clicked the Channel Balance does not display, only if you maximize the window or e.g. click on settings and then it shows up. If someone knows how to fix it I would be grateful.

### CODE REVIEW

File: src/app/components/ChannelList/custom-progress.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.progress {
+    height: 5px;
+    opacity: 0.7 !important;
+}
+
+.progress-bar {
+    background-color: #7642FF !important;
```
Comment (wbobeirne) [inline src/app/components/ChannelList/custom-progress.css]: Should be a LESS variable

File: src/app/components/ChannelList/ChannelRow.tsx [hunk 2]
```diff
@@ -4,6 +4,8 @@ import Identicon from 'components/Identicon';
 import Unit from 'components/Unit';
 import { ChannelWithNode } from 'modules/channels/types';
 import './ChannelRow.less';
+import {Progress} from 'reactstrap';
+import './custom-progress.css';
```
Comment (wbobeirne) [inline src/app/components/ChannelList/ChannelRow.tsx]: All styles in src should use LESS rather than CSS.

File: package.json [hunk 3]
```diff
@@ -49,11 +50,11 @@
     "babel-polyfill": "6.26.0",
     "bip39": "2.5.0",
     "bn.js": "4.11.8",
+    "bootstrap": "^4.1.3",
     "classnames": "^2.2.6",
     "clean-webpack-plugin": "^0.1.19",
     "copy-webpack-plugin": "4.6.0",
     "crypto-js": "3.1.9-1",
-    "css-loader": "^1.0.0",
```
Comment (wbobeirne) [inline package.json]: Why was this removed? It's still being used in the webpack config.

File: src/app/components/ChannelList/custom-progress.css (hunk 1 above)
Comment (PatrickLemke) [reply src/app/components/ChannelList/custom-progress.css]: fixed

File: src/app/components/ChannelList/ChannelRow.tsx (hunk 2 above)
Comment (PatrickLemke) [reply src/app/components/ChannelList/ChannelRow.tsx]: fixed

File: package.json (hunk 3 above)
Comment (PatrickLemke) [reply package.json]: Dump mistake on my part, fixed.

**Comment by PatrickLemke** [comment]
Thanks for giving such detailed feedback, in hindsight this was a very embarrassing PR. I didn't follow any of your guidelines and added unnecessary code, so shame on me.
Thanks for being so nice, hopefully those changes make more sense now. Also let me know if want any other changes to the design, etc.

**Comment by wbobeirne** [comment]
No worries man! Appreciate you taking the feedback, I'll take another look later. I'm also working on contributing guidelines (#45) which should make this kind of stuff more explicit.

**Comment by wbobeirne** [comment]
I made a couple of changes that I've PR'd against your PR over here: https://github.com/PatrickLemke/joule-extension/pull/2

Hope you don't mind! I thought that would be easier than telling you some of my concerns. Let me know if you have any thoughts.
